data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -178,8 +178,8 @@ amenity-charging_station                            # icon z16- (also has captio
 amenity-vending_machine-fuel                        # icon z17- (also has caption(optional) z17-)
 === 3150
 
-railway-station-subway-moscow                       # icon z11- (also has caption(optional) z12-)
-railway-station-subway-spb                          # icon z11- (also has caption(optional) z12-)
+railway-station-subway-moscow                       # icon z11-15
```

Comment:
Looks like making veh style captions z16- was a bit too much (no captions for Moscow subway now).
Let's make them z15-, it'll be big enough change from z12- already.
(and entrances are z15- by the way)

Reply:
> Looks like making veh style captions z16- was a bit too much (no captions for Moscow subway now).

moscow, kiev & spb etc all hide the main station icon at z16- and only show the entrances, so that's intentional. Will change to z15 though 👍️